apps/mobile/src/screens/chat/chat.tsx

```diff
@@ -1,11 +1,57 @@
 import React from "react";
 
-import { ScreenWrapper, Text } from "~/libs/components/components";
+import { Checkbox, ScreenWrapper, View } from "~/libs/components/components";
+import { useState } from "~/libs/hooks/hooks";
+import { globalStyles } from "~/libs/styles/styles";
+
+const mockCategories = [
+	{
+		id: 1,
+		name: "Physical",
+	},
+	{
+		id: 2,
+		name: "Work",
+	},
+];
+
+type FormData = Record<string, boolean>;
 
 const Chat: React.FC = () => {
+	const [formData, setFormData] = useState<FormData>({
```

Comment:
I see you already create a form, let's use `react-hook-form` here and use `control` for Checkbox

Reply:
Not resolved